be/src/exprs/agg/aggregate.h

```diff
@@ -282,7 +283,12 @@ class AggregateFunctionStateHelper : public AggregateFunction {
 public:
     static constexpr bool pod_state() { return std::is_trivially_destructible_v<State>; }
 
-    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const final { new (ptr) State; }
+    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const final {
+        new (ptr) State;
+        init(ctx, ptr);
+    }
+
+    void init(FunctionContext* ctx, AggDataPtr __restrict ptr) const override {}
```

Comment:
virtual function in final create will effect inline call in batch_create

Reply:
I think we shouldn't change the aggregate framework if aggregate function couldn't works for it.
we should implements from AggregateFunction rather than AggregateFunctionHelper